src/midje/parsing/1_to_explicit_form/parse_background.clj

```diff
@@ -263,6 +263,11 @@
                   commons/union
                   (set (map name symbols))))
 
+(defn remove-midje-fact-symbols [symbols]
+  (alter-var-root #'at-least-one-string-with-this-name-must-be-present
+                  commons/difference
+                  (set (map name symbols))))
```

Comment:
Sort of hacky to include this just to be used in tests. Unlike `add-midje-fact-symbols`, I left it hidden here instead of exposing it in `midje.sweet`